src/server/config/kibana.yml

```diff
@@ -61,7 +61,7 @@ verify_ssl: true
 
 # If you would like to send the log output to a file you can set the path below.
 # This will also turn off the STDOUT log output.
-# log_file: ./kibana.log
+log_file: /var/log/probe/kibana.log
```

Comment:
I ran into some issues with permissions when testing this. The directory the log is being created in needs to be owned by nginx user. If I use `./kibana.log`, the log file gets created in `/usr/local/kibana-4.1.10-linux-x64/`. That might work OK, but I wonder if we need to configure log rotation somewhere in the configuration. I think the best thing to do is put in a user story to configure this log file, but include log rotation in the story. Also do we want to make this log available through the GUI?